### DISCUSSION TITLE
请问怎么把本地的指定文件夹挂成webdav？还是必须使用网盘？

### DISCUSSION THREAD

**Discussion by caliban511** [AlistGo/alist#5754] [Q&A]
如题。
 不想用网盘，想用本地文件夹，方便自己管理和备份。
 我用的是docker版alist，在配置webdav的时候被难住了，，，
 求大佬们帮我看看这是哪里没写对？
朋友给我讲了后面加 dav 的用法，但这样只能把 webdav 挂到本地存储的子目录下，，，
请问能不能挂到自己指定的任意目录？

![Screenshot_2023-12-28-15-21-53-645_org mozilla firefox](https://github.com/alist-org/alist/assets/13557387/500d37b9-94f8-41b3-bbf4-bf675ba203a7)

![Screenshot_2023-12-28-15-21-59-233_org mozilla firefox](https://github.com/alist-org/alist/assets/13557387/b6966c5d-90ec-45df-968e-2107eb01112f)

![Screenshot_2023-12-28-15-22-24-759_org mozilla firefox](https://github.com/alist-org/alist/assets/13557387/4136f915-ec0b-4858-ae1f-8283858df121)

**Comment by yybshin** [comment]


**Reply by caliban511** [reply]
谢谢回复，但是并不行，不能自己任意指定文件夹位置。

**Reply by caliban511** [reply]
不是，你没看清楚，我是说 指定文件夹。
比如 本地存储 位置是 /mnt/1 ，然后把 webdav 的位置指到 /home/2 ，，，

**Reply by caliban511** [reply]
谢谢回复。
我说的是已经映射入docker容器内的目录。
例如：
当本地存储的目录设置为 /mnt/Caliban/Local
那么 webdav 的目录能不能设置为 /mnt/Caliban/WebDav
就是让这两个文件夹互不统属，各是各的？

或者干脆说明白一点，我目前在用的是两块硬盘，一块硬盘目前用来下载数据存一些不是很重要的东西，另一块硬盘放一些需要长久保存和比较私人的数据。
所以我把本地存储设置为那块用来下载东西的硬盘上的某一个目录，开放给网内所有用户使用，
同时把 webdav 设置为另一块硬盘上的某个目录，用来同步重要数据和放我一些私人数据。
权限设置目录映射硬盘挂载都已经解决，所以不必纠结上面举的例子和截图示例，也不需要考虑我会不会映射和挂载的事，，，
单就说说这个设置 webdav 到本地指定目录的事，alist能不能做到？
不知道我这样表述是否清晰，，，🙈🙈🙈
